src/migrations/1667818838202_add-rentedDays-and-periodChosen-columns.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from "node-pg-migrate"
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+const tableName = "rentals"
+const rentedDaysColumn = "rentedDays"
```

Comment:
We're using snake case for the columns and tables, would you mind changing it?
```suggestion
const rentedDaysColumn = "rented_days"
```

Reply:
Updated!